index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en-gb">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="guess_the_film.css">
+    <title>Guess the Film</title>
+</head>
+<body>
+    <main>
+        <section id="start_screen">
+            <h1>Guess the Film</h1>
+            <div class="instruction_wrapper">
+                <h2>Aim:</h2>
+                <p>Guess as many film titles from the quotes provided within 30s. Each correct answer is worth 1 point.</p>
+            </div>
+            <div class="instruction_wrapper">
+                <h2>How to play:</h2>
+                <ul>
+                    <li>Press 'Start Game' to begin</li>
+                    <li>Read the quote displayed, enter your guess and click 'Submit'</li>
+                    <li>Pressing 'Hint' reveals 3 possible answers and the year the film was released</li>
+                    <li>Click 'Next' to move to the next questions</li>
+                    <li>Once a game is finished, click 'Start Again' to start a new game</li>
+                </ul>
+            </div>
+            <button id="start_button">Start Game</button>
```

Comment:
remove start button as it is considered 'unfinished functionality' at this point. Should be brought in on the story that it becomes functional

Reply:
done!